draft-nottingham-avoiding-internet-centralization.md

```diff
@@ -179,7 +179,7 @@ Centralization risk is strongest when it necessarily affects the entire Internet
 
 "Decentralization" is the process of identifying centralization risk in the functions of a protocol or application, followed by application of appropriate techniques to prevent or mitigate centralization.
 
-Decentralization does not require that a given function need be so widely distributed that other important factors, such as efficiency, resiliency, latency, or availability are sacrificed. In some cases, a function that is only available through a relatively small number of providers can be  effectively decentralized, given the appropriate circumstances (see, for example, the DNS).
+Decentralization does not require that a given function need be so widely distributed that other important factors, such as efficiency, resiliency, latency, or availability are sacrificed. In some cases, a function that is only available through a relatively small number of providers can be  effectively decentralized, given the appropriate circumstances (see, for example, the Domain Name System {{?RFC1035}}).
```

Comment:
This was swapped with later text to avoid using an undefined initialism. I also wonder if [RFC 8499](https://www.rfc-editor.org/rfc/rfc8499.html) would serve as a better entry point than RFC 1035.

Reply:
Unfortunately, the DNS specs are a bit of a mess, so there isn't any really good reference that's actually helpful to readers. 8499 is only about terminology and a BCP, not a definitional document.

Reply:
That's true, but it does provide numerous links into the various definitional documents and itself serves as something of an overview. However, I don't see a particular problem with using RFC 1035—I was just musing about an alternative.